.changes/0.0.0.md

```diff
@@ -0,0 +1,4 @@
+## Previous Releases
+For information on prior major and minor releases, see their changelogs:
+- [1.1](https://github.com/dbt-labs/dbt-snowflake/blob/1.1.latest/CHANGELOG.md)
```

Comment:
This should not be here since it _is_ 1.1.

Reply:
great catch!